pkg/controllers/updaterun/controller_integration_test.go

```diff
@@ -67,102 +65,21 @@ var _ = Describe("Test the clusterStagedUpdateRun controller", func() {
 	})
 
 	Context("Test deleting a clusterStagedUpdateRun", func() {
-		It("Should delete the clusterStagedUpdateRun if it's not started yet", func() {
+		It("Should delete the clusterStagedUpdateRun without any clusterApprovalRequests", func() {
```

Comment:
can we add another case with cluserApprovalRequest (and not approved, thus stuck)?

Reply:
There is one with clusterApprovalRequests. But I'm not sure why deletion will get stuck if the requests are not approved?

Reply:
added more tests.